tests/accessor/accessor_common.h

```diff
@@ -485,7 +485,7 @@ void read_write_zero_dim_acc(AccT testing_acc, ResultAccT res_acc) {
     res_acc[0] = value_operations::are_equal(acc_ref, other_data);
   }
   if constexpr (AccessMode != sycl::access_mode::read) {
```

Comment:
Shouldn't we apply the fix to the code in `AccessMode != sycl::access_mode::write` branch?

Reply:
I didn't think it mattered in that branch because the initialization `acc_ref` would still read from the accessor and no reference is necessary for testing of equality.